src/language-css/print/parenthesized-value-group.js

```diff
@@ -155,4 +164,18 @@ function shouldBreakList(path) {
   );
 }
 
+/**
+ * @template {*} T
+ * @param {T[]} xs
+ * @param {number} chunkSize
+ * @returns {T[][]}
+ */
+function chunk(xs, chunkSize) {
```

Comment:
What is `xs`? `array` should be a better name.

Maybe also `chunkSize` -> `size`

Reply:
Thank you, I'll rename xs by array.
For size, I wanted to avoid confusion between "each chunk has n elements" vs
"result has n chunks". How about `eachSize`?

Reply:
It's common to name as size.

https://github.com/ryancole/chunk/blob/39631771da7e5d3eb8f43d6ebcd52905c02b46ed/src/chunk.js#L4

https://github.com/lodash/lodash/blob/6a2cc1dfcf7634fea70d1bc5bd22db453df67b42/src/chunk.ts#L22